mmv1/templates/terraform/custom_import/apigee_organization_keyvaluemaps_entries.go.tmpl

```diff
@@ -0,0 +1,18 @@
+config := meta.(*transport_tpg.Config)
+
+// current import_formats cannot import fields with forward slashes in their value
+if err := tpgresource.ParseImportId([]string{
+		"(?P<org_keyvaluemap_id>.+)/entries/(?P<name>.+)",
+		"(?P<org_keyvaluemap_id>.+)/(?P<name>.+)",
+	}, d, config); err != nil {
+		return nil, err
+	}
+
+// Replace import id for the resource id
+id, err := tpgresource.ReplaceVars(d, config, "{{org_keyvaluemap_id}}/entries/{{name}}")
```

Comment:
I think the curly braces here need to be fixed, since it is a template?

Reply:
fixed , thanks